src/utils/parson_json_utils/src/parson_json_utils.c

```diff
@@ -15,6 +15,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <time.h>
+#include <unistd.h>
```

Comment:
can remove? don't see any change in here requiring this (and it would need to be aducpal/unistd.h anyways).

Reply:
Removed